src/erlcloud_iam.erl

```diff
@@ -74,15 +88,15 @@ configure(AccessKeyID, SecretAccessKey, Host) ->
 %
 % Users API
 %
--spec(get_user/0 :: () -> proplist()).
+-spec(get_user/0 :: () -> {ok, proplist()}).
 get_user() -> get_user([]).
--spec(get_user/1 :: (string() | aws_config()) -> proplist()).
+-spec(get_user/1 :: (string() | aws_config()) -> {ok, proplist()}).
```

Comment:
this interface change from `proplists()` to `{ok, proplists()}` will brake a lot of things!